Ginger/Ginger/ALM/ALMConnectionPage.xaml.cs

```diff
@@ -410,6 +410,11 @@ private void StyleRadioButtons()
         private void SaveALMConfigs()
         {
             ALMIntegration.Instance.SyncConfigurations();
+            Ginger.SolutionGeneral.Solution lastSavedSolution = Ginger.SolutionGeneral.Solution.LoadSolution(App.UserProfile.Solution.FilePath, false);
+            if (!(App.UserProfile.Solution.ALMServerURL == lastSavedSolution.ALMServerURL && App.UserProfile.Solution.ALMDomain == lastSavedSolution.ALMDomain && App.UserProfile.Solution.ALMProject == lastSavedSolution.ALMProject))
+            {
+                App.UserProfile.Solution.ExternalItemsFields.Clear();
+            }
             App.UserProfile.SaveUserProfile();
```

Comment:
we agreed this approch won't work no? 
because what if:
1- user set QC A + fields and close Ginger
2- open Ginger shift to QC B and load it fields
when he will do save above code will clear all of his QC B fields which were loaded in valid way.
i think we should clear the fields on cache as soon as user changes the URL/domain/Project and not on save so if user didn't loaded new fields he will have empty list on save and if he handeld new list so we will save the valid list.